connectors/http/rest/element-templates/http-json-connector.json

```diff
@@ -352,7 +352,7 @@
     "constraints" : {
       "notEmpty" : true,
       "pattern" : {
-        "value" : "^(=|(http://|https://|secrets|\\{\\{).*$)",
+        "value" : "^(http://|https://|secrets|\\{\\{).*$",
```

Comment:
Would that still work with FEEL expressions?

Reply:
Yes thank you, the HttpCommonRequest had `@Pattern` instead of `@SecretPattern`